src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx

```diff
@@ -18,11 +18,9 @@ freshnessValidatedDate: never
 
 import queriesnrqlFunnelNRQL from 'images/queries-nrql_screenshot-crop_funnel-NRQL.webp'
 
-With [NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql), you can use funnels to evaluate sets of related actions. The actions generally share a relationship via an identifier such as a user ID or session ID.
+With [NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql), you can use funnels to evaluate sets of related actions. The actions generally share a relationship with an identifier, such as a user ID or session ID.
```

Comment:
I think replacing "via" actually muddies the waters here a bit. I do approve of the comma tho